randovania/games/dread/pickup_database/pickup-database.json

```diff
@@ -517,6 +538,7 @@
             "broad_category": "misc",
             "model_name": "powerup_doublejump",
             "offworld_models": {
+                "am2r": "Space Jump",
```

Comment:
am2r high jump uses the spin boost model in dread, so maybe it should be the other way around for spin boost in am2r? not a huge deal since most people use progressive spin

Reply:
Whoops, good catch